wp-includes/widgets/class-wp-widget-media.php

```diff
@@ -54,8 +54,7 @@ public function __construct( $id_base = '', $name = '', $widget_options = array(
 
 		parent::__construct(
 			$id_base,
-			/* translators: placeholder is the id base of the widget */
-			$name ? $name : sprintf( __( 'Media: %s' ), $id_base ),
+			$name,
```

Comment:
Would you want to go as far as removing the empty string fallbacks in the method's arguments list?